app/controllers/aws_ip_ranges_webhooks_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AwsIpRangesWebhooksController < ActionController::Base
+  skip_before_action :verify_authenticity_token
+
+  def update_aws_ip_ranges
+    RemoteIpTrustedProxiesService.configure_trusted_proxies(RemoteIpTrustedProxiesService.load_current_aws_ip_ranges)
+    Rails.logger.info("LOGGING AWS SNS SUBSCRIPTION CONFIRMATION FOR UPDATE_AWS_IP_RANGES WEBHOOK: #{params}")
```

Comment:
An alternative to logging this out so that we can confirm ownership manually would be for the webhook controller to visit the confirmation URL automatically. I didn't implement that because this work already got blocked by enough other things and I'd like to not make it any more complex.